integrationkey/queries.sql

```diff
@@ -0,0 +1,20 @@
+-- name: GetServiceID :one
```

Comment:
Since all queries get put into the same package, let's prefix these with something like `IntKey`, like `IntKeyServiceID` `IntKeyCreate` `IntKeyFindOne` `IntKeyFindByService` etc...

Also, run this file through `pgFormatter`